### ISSUE TITLE
Duplicate RGB values for elements in ElementColorSchemes.yaml

### ISSUE THREAD

**Issue by tsmathis** [materialsproject/crystaltoolkit#359]
Two sets of two elements share duplicate RGB values in the VESTA color scheme in `ElementColorSchemes.yaml`. This leads to visual ambiguity in the rendered structures for materials with these elements in their chemical system.

VESTA:
F and N : [176, 185, 230]
Sr and Zr: [0, 255, 0]

Some examples: [mp-770419](https://next-gen.materialsproject.org/materials/mp-770419?formula=Sr2Zr7O16), [mp-601344](https://next-gen.materialsproject.org/materials/mp-601344?_sort_fields=-nsites&elements=F%2C%20N&nelements_max=5&nelements_min=0)

**Comment by mkhorton** [comment]
This is true, but not our choice: these are reference color schemes determined by others.

I completely support the addition of a new Crystal Toolkit color scheme however, which we can make a default. Happy to discuss.

**Comment by mkhorton** [comment]
Aside: this partly motivated the addition of the “Accessible” color scheme. This used a reference color scheme from literature suitable for people with color blindness, and constrained atoms to be their “regular” color if possible (i.e. close to CPK, oxygen red etc.) 

This scheme is not fully successful yet since one of the colors in that scheme is white, so to really reach accessibility goals, it’d probably be better off switching to a black/darker viewport background when that scheme is enabled, or simply changing the white atom color to something else, but this was on the backlog.

**Comment by tsmathis** [comment]
Good to know. Color schemes came up in a MP update call a few weeks ago in a related context, I can bring it up again in the next call to see what everyone thinks.
